charts/fission-all/templates/deployment.yaml

```diff
@@ -49,9 +49,104 @@ rules:
   resources:
   - packages
   verbs:
+  - '*'
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - rolebindings
+  verbs:
+  - '*'
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: fission-cr-admin
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  - pods
+  - secrets
+  - services
+  - serviceaccounts
```

Comment:
Creating SA might still be a security concern. Once user has installed fission, the service accounts would exist by default. The only reason we require this permission is because the controller will create if they don't already exist, right? If so,  two scenarios where this helps is:
* Getting into a state where service accounts are deleted 
*  Incomplete installation 
Can't we just give an error to the user  through when such a thing happens?

Reply:
https://github.com/fission/fission/issues/1928